Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```diff
@@ -34,6 +33,15 @@
     sprite: Clothing/OuterClothing/Armor/security_slim.rsi
   - type: Clothing
     sprite: Clothing/OuterClothing/Armor/security_slim.rsi
+  - type: Armor #Based on /tg/ but slightly compensated to fit the fact that armor stacks in SS14.
```

Comment:
Remove this entire part, don't touch the Wizden vest.

Reply:
Commented out the modifiers but not the sprites to ensure that compiling runs smoohthly.